app/library/Takajou/Def/Db/ClusterType.php

```diff
@@ -5,4 +5,5 @@ class ClusterType {
     const NONE   = 'none';
     const MASTER = 'master';
     const SLAVE  = 'slave';
+    const sidecitest = 'sidecitest';
```

Comment:
[code_sniffer] Class constants must be uppercase; expected SIDECITEST but found sidecitest